features/adres.feature

```diff
@@ -3,88 +3,58 @@
-      | fields              | burgerservicenummer,verblijfplaats.woonplaats,verblijfplaats.gemeenteVanInschrijving |
-      Dan heeft de persoon met burgerservicenummer '<burgerservicenummer>' alleen de volgende 'verblijfplaats' gegevens
-      | naam                                 | waarde        |
-      | type                                 | Adres         |
-      | gemeenteVanInschrijving.code         | 0518          |
-      | gemeenteVanInschrijving.omschrijving | 's-Gravenhage |
-      | woonplaats                           | <woonplaats>  |
-
-      Voorbeelden:
-      | burgerservicenummer | woonplaatsnaam | woonplaats    |
-      | 999992922           | Scheveningen   | Scheveningen  |
-      | 999990639           |                | 's-Gravenhage |
-
-  Rule: Voor een binnenlands adres wordt adresregel1 samengesteld conform NEN 5825:2002.
-
-    # Veld adresregel1 wordt samengesteld uit locatiebeschrijving + korteNaam + aanduidingBijHuisnummer + huisnummer + huisletter + huisnummertoevoeging
-    # De aanduidingBijHuisnummer waarde "tegenover" (11.50="to") wordt in "adresregel1" opgenomen als "t/o", waarde "bij" (11.50="by") wordt in "adresregel1" opgenomen als "bij"
-    # Tussen kortenaam en huisnummer wordt een spatie opgenomen
-    # Tussen het huisnummer en de huisnummertoevoeging (wat in de NEN een samenstelling is van huisletter en huisnummertoevoeging, niet gescheiden van elkaar) moet een koppelteken ("-") worden geplaatst indien de huisnummertoevoeging met een cijfer begint, een spatie in alle andere gevallen
+  Rule: Voor een binnenlands adres wordt adresregel1 samengesteld conform NEN 5825:2002
+    - Veld adresregel1 wordt samengesteld uit korteNaam + aanduidingBijHuisnummer + huisnummer + huisletter + huisnummertoevoeging
+    - De aanduidingBijHuisnummer (11.50) waarde "to" (tegenover) wordt in "adresregel1" opgenomen als "t/o", waarde "by" wordt in "adresregel1" opgenomen als "bij"
+    - Tussen kortenaam en aanduidingBijHuisnummer of huisnummer wordt een spatie opgenomen
+    - Tussen aanduidingBijHuisnummer en huisnummer wordt een spatie opgenomen
+    - Tussen het huisnummer en de huisnummertoevoeging (wat in de NEN een samenstelling is van huisletter en huisnummertoevoeging, niet gescheiden van elkaar) moet een koppelteken ("-") worden geplaatst indien de huisnummertoevoeging met een cijfer begint, een spatie in alle andere gevallen
 
     Abstract Scenario: adresregel1 voor een binnenlands adres met <omschrijving>
       Gegeven het systeem heeft een persoon met de volgende gegevens
       | naam                | waarde                |
       | burgerservicenummer | <burgerservicenummer> |
       En de persoon heeft de volgende 'verblijfplaats' gegevens
       | naam                              | waarde                      |
-      | straatnaam (11.10)                | <straatnaam>                |
+      | straat (11.10)                    | <straatnaam>                |
```

Comment:
als ik in LO kijk zie ik wel dat voor element 11.10 straatnaam wordt gebruikt en niet straat. Wat mij betreft prima om straat te gebruiken ipv straatnaam, maar just checking

Reply:
maakt mij ook niet uit of we het straat of straatnaam noemen

Reply:
ik heb het teruggewijzigd naar straatnaam (11.10)